src/accelerate/accelerator.py

```diff
@@ -1634,6 +1634,9 @@ def save_state(self, output_dir: str = None):
         Args:
             output_dir (`str` or `os.PathLike`):
                 The name of the folder to save all relevant weights and states.
+            save_model_func_kwargs (`dict`, *optional*):
+                Additional keyword arguments for saving model which can be passed to the underlying save function, such
+                as optional arguments for `DeepSpeed`'s `save_checkpoint` function.
```

Comment:
```suggestion
                as optional arguments for DeepSpeed's `save_checkpoint` function.
```

either  `deepspeed` (package) or  DeepSpeed (name / no backquotes)